python_modules/dagster/dagster/core/storage/migration/utils.py

```diff
@@ -259,3 +260,32 @@ def create_schedule_secondary_index_table():
             db.Column("create_timestamp", db.DateTime, server_default=db.text("CURRENT_TIMESTAMP")),
             db.Column("migration_completed", db.DateTime),
         )
+
+
+def create_instigators_table():
+    if not has_table("instigators") and not has_table("jobs"):
+        # not a schedule storage db
+        return
+
+    if (
+        has_table("instigators")
+        and has_column("job_ticks", "selector_id")
+        and has_column("jobs", "selector_id")
+    ):
+        # already migrated
+        return
+
+    op.create_table(
+        "instigators",
+        db.Column("id", db.Integer, primary_key=True, autoincrement=True),
+        db.Column("selector_id", db.String(255), unique=True),
+        db.Column("repository_name", db.Text),
```

Comment:
i feel like this should be repository_selector_id not repository_name. If two repositories in different locations have the same name we will want to be able to filter them separately